lib/puppet/provider/pam/augeas.rb

```diff
@@ -45,6 +45,35 @@ def self.target(resource = nil)
     end
   end
 
+  def self.position_path (position, type)
+    placement, identifier, value = position.split(/ /)
+    key = !!value
+    if PAM_POSITION_ALIASES[key].has_key? identifier
+      expr = PAM_POSITION_ALIASES[key][identifier]
+      expr = key ? expr % [type, value] : expr % [type]
+    else
```

Comment:
I might have liked to keep the comment that was in the original piece of code here.